### DISCUSSION TITLE
Vortex should have optional flags for ban command.

### DISCUSSION THREAD

**Discussion by XynoxTheDev** [jagrosh/Vortex#94] [Questions] [answered]
When a Mod is using the ban command there should be an optional flag that the recent messages are to delete or not. Else those messages can be stored in logs for future help for some other Mod.

**Comment by jagrosh** [comment, marked as answer]
I believe the `silentban` is the command you're looking for, which bans without deleting messages.

**Reply by XynoxTheDev** [reply]
Yes, is there any command such that in vortex or flag with hackban?

**Reply by jagrosh** [reply]
> Yes, is there any command such that in vortex or flag with hackban?

`>>silentban` is identical to `>>ban` except that it does not delete messages. As per [this page on the wiki](https://github.com/jagrosh/Vortex/wiki/Rarely-Asked-Questions#what-is-hackban), "hackban" is just a legacy alias for `ban` and is identical to `>>ban`

**Reply by XynoxTheDev** [reply]
Thanks, @jagrosh!
